### DISCUSSION TITLE
PCB Build

### DISCUSSION THREAD

**Discussion by dpwe** [shorepine/tulipcc#26] [Show and tell]
Just wanted to share that I successfully built a PCB-based Tulip!
![tulip-build](https://user-images.githubusercontent.com/552145/224814405-4c3a1d85-2d2e-4798-b778-d1bed35a37c1.jpg)
This was all hand-soldered, and the FPC and the USB-C connectors were pretty challenging.  Would not have been able to do it without ChipQuick Flux and desoldering braid.

I managed to connect to the wrong connector on the back of the display the first time (LVDS instead of RGB).  Bad mistake - after I fixed it, the screen *mostly* worked, but was somehow sufficiently out of spec that I spent a long time looking for problems in software (Brian too), when all I really needed was another screen (sad face).

Oh, and I blew out the small 6 pin transistor package by bridging the ESP32_EN pin to +3v, which burned out the transistors and the USB chip.  Luckily I had a second USB chip, but I had to replace the transistors with discretes (until my next DigiKey comes in).

**Comment by bwhitman** [comment]
Nice work dan !! thanks for all the help finding issues with the build process and documentation, it'll be a lot easier for the next person to build a Tulip! Tulip no2 is online!
